docs/internals/security.txt

```diff
@@ -38,6 +43,152 @@ action to be taken, you may receive further followup emails.
 
 .. _our public Trac instance: https://code.djangoproject.com/query
 
+Reporting guidelines
```

Comment:
Thought: Should these "guidelines" have a bit of text explaining that if someone is unsure whether an issue meets the guidelines, to err on the side of caution and submit "responsibly" anyway? In that it's better to fail safe. 

It also makes the "guidelines" feel slightly less like rules (which they're not).

Reply:
We have this paragraph within "How does Django evaluate a report"

> If you are unsure whether your finding meets these criteria, please still report
it :ref:`privately by emailing security@djangoproject.com
<reporting-security-issues>`. The security team will review your report and
recommend the correct course of action.

I think it's ok to keep that there.
I would be very pleasantly surprised is people have read these guidelines.
I expect that we link to points in these guidelines to explain why we have concluded that their report is not valid/they need to take these into account and send us an update